spec/features/adventures/user_views_adventure_spec.rb

```diff
@@ -0,0 +1,77 @@
+require "rails_helper"
+
+feature "user views an of their adventures", %(
+  As a last minute traveler or traveler who's stay is unexpectedly extended,
+  I want to view my adventures
+  So that I can explore adventures on the fly.
+) do
+  # Acceptance Criteria
+  # [] I must be signed in to view my private adventures
+  # [] If my adventure includes a link and I click it, I am taken to the link
+  #    address
+  # [] I can view a list of the adventures associated with each bucket_list
+  # [] I can see an icon telling me if my adventure is public or private
+  # [] I can click an icon link to switch any on my adventures between public
+  #    and private
+  # [] I can click a link to edit my adventure
+
+  scenario "authenticated user successfully views list of their adventures " \
+  "associated w/ a particular bucket lists by navigating to the bucket" \
+  " list show page" do
+    user = FactoryGirl.create(:user)
+
+    # bucket_list_1
+    bucket_list_1 = FactoryGirl.create(:bucket_list, user_id: user.id)
+    adventure_1 = FactoryGirl.create(
+      :adventure
+    )
+    bucket_list_adventure_1 = FactoryGirl.create(
```

Comment:
Useless assignment to variable - `bucket_list_adventure_1`. Did you mean `bucket_list_adventure_2`?